qiita_pet/handlers/study_handlers/sample_template.py

```diff
@@ -57,3 +62,71 @@ def post(self):
         else:
             raise HTTPError(400, 'Unknown sample template action: %s' % action)
         self.write(result)
+
+
+def _build_sample_summary(study_id, user_id):
```

Comment:
I would like to hear from others @antgonza @ElDeveloper 

In my opinion, using pandas here is an overkill. The arguments of using pandas was that it handles the merge of the series for us and it can generate the HTML for us. I think this can be simplified by a simple set of dictionaries and just generate the HTML ourselves.

Also, here we are generating the HTML, then parsing it with xml.etree and then writing back the string. This just looks like patches to fix something that doesn't work in the way that we actually want.

However, these are my ideas and I leave it open to other devs to decide if this is ok as it is.